src/Expectation.php

```diff
@@ -118,6 +118,32 @@ public function dd(mixed ...$arguments): void
         exit(1);
     }
 
+    /**
+     * Dump the expectation value when the result of the condition is truthy.
+     *
+     * @param  bool  $boolean
+     * @return never
+     */
+    public function ddWhen($boolean, mixed ...$arguments): void
+    {
+        if (! $boolean) {
+            return;
+        }
+
+        $this->dd($this->value, ...$arguments);
+    }
+
+    /**
+     * Dump the expectation value when the result of the condition is falsy.
+     *
+     * @param  bool  $boolean
+     * @return never
+     */
+    public function ddUnless($boolean, mixed ...$arguments): void
```

Comment:
```suggestion
    public function ddUnless(bool $boolean, mixed ...$arguments): void
```

Reply:
> 

Hey! @faissaloux . Thank you, but according to the all code style of Pest I think this can't be inserted here.